File: porting-console-app-rhill34/visibility/Address.cs
using System;
namespace visibility
{
    public class Address
    {

        public Address(int number, string line2, string city,
            string state, string zipcode)
        {
            Number = number;
            Street = line2;
            City = city;
            State = state;
            Zip = zipcode;
        }

        public int Number { get; set; }
        public string Street { get; set; }
        public string City { get; set; }
        public string State { get; set; }
        public string Zip { get; set; }

        public override string ToString()
        {
            return "Address: " + Environment.NewLine +
                +Number+" "+ Street + Environment.NewLine + City +
                ", "+State+", "+ Environment.NewLine + Zip;
        }
    }
}

File: porting-console-app-rhill34/javaFiles/visibility/Address.java
package visibility;

public class Address {
    private int number;
    private String street;
    private String city;
    private String state;
    private String zipcode;


    public Address(int number, String line2, String city, String state, String zipcode) {
        this.number = number;
        this.street = line2;
        this.city = city;
        this.state = state;
        this.zipcode = zipcode;
    }


    @Override
    public String toString() {
        return "Address:  \n" + number + " " + street + "\n" + city + ", " + state + "\n       "+zipcode;

    }

}